sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -1,33 +1,36 @@
-# Authentication Events Trigger for Azure Functions client library for .NET
+# Authentication events trigger for Azure Functions
 
-Authentication Event Trigger for Azure Functions handles all the backend processing, (e.g. token/json schema validation) for incoming Http requests for Authentication events. And provides the developer with a strongly typed, versioned object model to work with, meaning the developer need not have any prior knowledge of the request and response json payloads.
+The authentication events trigger for Azure Functions allows you to implement a custom extension to handle Azure Active Directory (Azure AD) authentication events.  The authentication events trigger handles all the backend processing for incoming HTTP requests for Azure AD authentication events and provides the developer with:
 
-This project framework provides the following features:
-
-* Token validation for securing the API call
-* Object model, typing, and IDE intellisense
-* Inbound and outbound validation of the API request and response schemas
-* Versioning
-* No need for boilerplate code.
+- Token validation for securing the API call
+- Object model, typing, and IDE intellisense
+- Inbound and outbound validation of the API request and response schemas
+- Versioning
 
 ## Getting started
 
 ### Install the package
 
+Install the authentication events trigger for Azure Functions with [NuGet](https://www.nuget.org/):
+
 ```dotnetcli
 dotnet add package Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents --prerelease
 ```
 
 ### Prerequisites
 
-* [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) for Windows **OR** [Visual Studio Code >= 1.61](https://code.visualstudio.com/download)
-* [Dotnet core 3.1](https://dotnet.microsoft.com/download/dotnet/3.1)
-* [Azure function tools 3.30](https://github.com/Azure/azure-functions-core-tools)
-* [Nuget](https://docs.microsoft.com/nuget/install-nuget-client-tools)
-* [Azure Function Core Tools](https://github.com/Azure/azure-functions-core-tools#installing)
-* If using Visual Studio Code the following extensions:
-  * [ms-azuretools.vscode-azurefunctions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
-  * [ms-dotnettools.csharp](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+- **Azure Subscription:**  To use Azure services, including Azure Functions, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+- [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) or later for Windows **OR** [Visual Studio Code >= 1.61](https://code.visualstudio.com/download) or later
```

Comment:
This isn't a prerequisite; other editors such as Rider, Sublime Text, and notepad can be used.

Reply:
Good point, removing.